apps/src/templates/sectionProgressV2/MoreDetailsDialog.jsx

```diff
@@ -1,27 +1,118 @@
+            <StrongText>{i18n.noOnlineWork() + ': '}</StrongText>
+            {i18n.progressLegendDetailsNoOnlineWork()}
+          </BodyThreeText>
+        </div>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.IN_PROGRESS} />
+          <BodyThreeText>
+            <StrongText>{i18n.inProgress() + ': '}</StrongText>
+            {i18n.progressLegendDetailsInProgress()}
+          </BodyThreeText>
+        </div>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.SUBMITTED} />
+          <div>
+            <BodyThreeText>
+              <StrongText>{i18n.submitted() + ': '}</StrongText>
+            </BodyThreeText>
+            <SafeMarkdown
+              className={styles.firstMarkdown}
+              markdown={i18n.progressLegendDetailsSubmittedForLessons()}
+            />
+            <SafeMarkdown
+              markdown={i18n.progressLegendDetailsSubmittedForLevels()}
+            />
+          </div>
+        </div>
+        {hasValidation && validatedLevelsContent()}
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.KEEP_WORKING} />
+          <BodyThreeText>
+            <StrongText>{i18n.markedAsKeepWorking() + ': '}</StrongText>
+            {i18n.progressLegendDetailsKeepGoing()}
+          </BodyThreeText>
+        </div>
+        <Heading6>{i18n.teacherActions()}</Heading6>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.NEEDS_FEEDBACK} />
+          <BodyThreeText>
+            <StrongText>{i18n.needsFeedback() + ': '}</StrongText>
+            {i18n.progressLegendDetailsNeedsFeedback()}
+          </BodyThreeText>
+        </div>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.VIEWED} />
+          <BodyThreeText>
+            <StrongText>{i18n.viewed() + ': '}</StrongText>
+            {i18n.progressLegendDetailsViewed()}
+          </BodyThreeText>
+        </div>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.FEEDBACK_GIVEN} />
+          <BodyThreeText>
+            <StrongText>{i18n.feedbackGiven() + ': '}</StrongText>
+            {i18n.progressLegendDetailsFeedbackGiven()}
+          </BodyThreeText>
+        </div>
+        <Heading6>{i18n.levelTypes()}</Heading6>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.ASSESSMENT_LEVEL} />
+          <BodyThreeText>
+            <StrongText>{i18n.assessmentLevel() + ': '}</StrongText>
+            {i18n.progressLegendDetailsAssessmentLevels()}
+          </BodyThreeText>
+        </div>
+        <div className={styles.item}>
+          <ProgressIcon itemType={ITEM_TYPE.CHOICE_LEVEL} />
+          <BodyThreeText>
+            <StrongText>{i18n.choiceLevel() + ': '}</StrongText>
+            {i18n.progressLegendDetailsChoiceLevels()}
+          </BodyThreeText>
         </div>
-      </div>
-      <div>
-        <Button
```

Comment:
I removed this button at the bottom in favor for the X at the top of the pop-up which seemed more intuitive.